examples/generator/bernstein_vazirani.txt

```diff
@@ -0,0 +1,25 @@
+// Implementation of a Bernstein-Vazirani algorithm
+
+x 6
+h 0
+h 1
```

Comment:
I'll leave it for now, since this is really just a toy example and I'd have to look up/try out how to force the line break in a platform agnostic way.